src/sentry/templates/sentry/partial/preload-data.html

```diff
@@ -47,18 +47,33 @@
       return host + '/api/0/organizations/' + slug + suffix;
     }
 
-    var preloadPromises = {orgSlug: slug};
+    var preloadPromises = { orgSlug: slug };
     window.__sentry_preload = preloadPromises;
 
     if (!slug) {
       return;
     }
-
-    preloadPromises.organization = promiseRequest(makeUrl('/?detailed=0'));
-    preloadPromises.projects = promiseRequest(
-      makeUrl('/projects/?all_projects=1&collapse=latestDeploys&collapse=unusedFeatures')
-    );
-    preloadPromises.teams = promiseRequest(makeUrl('/teams/'));
+    /*
+     We don't want to preload data for _admin for 2 reasons:
+     1. We don't use the data because _admin looks at all orgs and aggregated
+        information, rather than your specific org.
+     2. When you go through the staff flow and activate U2F when you have an
+        active superuser session, there's a race condition that occurs. Both
+        the preflight requests and the GET to your U2F authenticators contain
+        a response session cookie. The preflight requests are sent first but
+        are much slower than the authenticator GET, so we end up storing the
+        slowest preflight request's session cookie on the client. This causes
+        the subsequent U2F validation request to fail because the session
+        cookie is missing the U2F state that was in the authenticator GET
+        response session cookie.
+    */
+    if (!window.location.pathname.startsWith("/_admin")) {
```

Comment:
Since `_admin` is in `getsentry`, is it possible to move this to that repo?

Reply:
Another approach you could take is to make running preloaders part of `window.__initialData`. That would let you push the conditions under which we don't preload into the server. There are a few other places where skipping preloads could be useful (user settings).

This could be done separately/later as well.

Reply:
I don't think this can be moved into getsentry b/c this is used in other html templates in sentry

Reply:
I think `__initialData` could just have a flag that indicates if we should pre-load or not?

Reply:
If I put the flag in `__initialData`, what would be the best way in code to set it to False for the routes I wanted? I think we have access to the request object
It'd look something like:
```Python
def get_context(self) -> Mapping[str, Any]:
      return {
            ...
            "skipPreloads": skip for _admin url
      }
```

Reply:
I think you would need some kind of hook that get's set by getsentry here

Reply:
We have done 'hooks' like this before with a mutable list in django settings that getsentry can append into.

Reply:
That works as well, but it might be a little bit more indirection I think.

Either way, we have to pass things from the server to client I think since the preload request script is served by the server though

Reply:
> Curious what y'all would think about using a flag in the jinja template, that's set in getsentry where the template is rendered instead?

Eliding the entire preload block would work well too.